packages/2d/src/Components/LowLevelMouse.ts

```diff
@@ -6,6 +6,9 @@ import {
 import { Vector } from "../Models";
 import { useContext, useUpdate, useEntityTransforms } from "../Hooks";
 
+type Callback = (event: HexMouseEvent) => void;
+class CallbackSet extends Set<Callback> { /***/ }
```

Comment:
I'm anxious that extending builtins might not work in all browsers. Can we leave these as normal Sets instead? With the `Callback` type alias, the construction sites will just be `new Set<Callback>()` instead of `new Set<(event: HexMouseEvent) => void>()`.